frontend/mr-admin-flate/src/components/avtaler/AvtaleSkjemaContainer.tsx

```diff
@@ -327,6 +327,7 @@ export function AvtaleSkjemaContainer({
                   label={"Avtaleansvarlig"}
                   {...register("avtaleansvarlig")}
                   onClearValue={() => setValue("avtaleansvarlig", "")}
+                  description="Den som blir satt som ansvarlig vil få en notifikasjon."
```

Comment:
Er denne teksten nødvendig? Selv oppfatter jeg det mest som støy, men bryr meg egentlig ikke så mye om den er med eller ikke :shrug:

Reply:
Litt samma for meg også. Man vil jo fort lære det

Reply:
Husk at vi som utvikler dette vet at dette vil skje, ref. denne PRen, men de som tar i admin-flate for første gang og aldri har fått en notifikasjon i admin-flate før kan nok tenkes at det er greit å vite at den som blir satt som ansvarlig får beskjed om det.